### PR TITLE
OLS-114: Check for all counters in metrics

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Check for all counters in metrics

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [x] Integration tests

## Related Tickets & Documents

- Related Issue #[OLS-114](https://issues.redhat.com//browse/OLS-114)

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: tests/integration/test_metrics.py [hunk 1]
```diff
@@ -36,8 +36,16 @@ def test_metrics():
...
+    expected_counters = (
+        "rest_api_calls_total",
+        "llm_calls_total",
+        "llm_calls_failures_total",
+        "llm_validation_errors_total",
+    )
+
+    # check if all counters are present
+    for expected_counter in expected_counters:
+        assert f"{expected_counter} " in response_text
```
Comment (onmete) [inline tests/integration/test_metrics.py:48]: why not just `assert expected_counter in response_text`? Is the extra space important?

Comment (tisnik) [reply tests/integration/test_metrics.py:48]: I would like to be precise, but depends, not having strong opinion there.

Comment (onmete) [reply tests/integration/test_metrics.py:48]: ok

**Pull request merged**